project/Vagrantfile

```diff
@@ -53,30 +63,31 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         end
         (ip = /^\s*inet .*?(\d+\.\d+\.\d+\.\d+)\s+/.match(result)) && ip[1]
     end
-    
+
     # Customize the virtual machine environments
     config.vm.provider :virtualbox do |vb|
         vb.customize ["modifyvm", :id, "--nictype1", "virtio"]
         vb.customize ["modifyvm", :id, "--nictype2", "virtio"]
         #vb.gui = true # for debugging
     end
-    
+
     config.vm.define mastername do |master|
         master.vm.box = "chef/centos-6.5"
         master.vm.provider "virtualbox" do |v|
             v.name = mastername
-            v.memory = 2048
+            v.memory = 6144
```

Comment:
6gb is too much if you want to create multiple vm's. 2gb per vm is better in that case. Someone already did a pull to make mem and cpu configurable. Time I look into merging that..